index.js

```diff
@@ -1,5 +1,8 @@
 module.exports = (nextConfig = {}) => {
   return Object.assign({}, nextConfig, {
+    publicRuntimeConfig: Object.assign({}, nextConfig.publicRuntimeConfig, {
+      nextImagesAssetPrefix: nextConfig.assetPrefix
```

Comment:
What's the difference between `assetPrefix` and `nextImagesAssetPrefix `?

Reply:
`nextConfig.assetPrefix` is a runtime configuration value which can be used to set asset prefix of js bundles and other asset files. According to official next.js docs it is only used internally by next and it is not accessible. That's why we need to expose it explicitly in `publicRuntimeConfig` so we can access it confidently on server side and client side too.

Since users could define their own public runtime configuration I thought we shouldn't call it simply "assetPrefix" because maybe we override a user defined config variable. So I ended up using a "namespaced" variable name.